tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/json/LabelSerializer.scala

```diff
@@ -1,14 +1,15 @@
 package com.linagora.tmail.james.jmap.json
 
-import com.linagora.tmail.james.jmap.model.{Color, DisplayName, Label, LabelCreationId, LabelCreationRequest, LabelCreationResponse, LabelGetRequest, LabelGetResponse, LabelId, LabelIds, LabelSetRequest, LabelSetResponse, UnparsedLabelId}
+import com.linagora.tmail.james.jmap.model.{Color, DisplayName, Label, LabelCreationId, LabelCreationRequest, LabelCreationResponse, LabelGetRequest, LabelGetResponse, LabelId, LabelIds, LabelPatchObject, LabelSetRequest, LabelSetResponse, LabelUpdateResponse, UnparsedLabelId}
 import org.apache.james.jmap.core.{Properties, SetError, UuidState}
 import org.apache.james.jmap.json.mapWrites
 import org.apache.james.jmap.mail.Keyword
 import play.api.libs.json.{Format, JsArray, JsError, JsObject, JsResult, JsString, JsSuccess, JsValue, Json, Reads, Writes, __}
 
 object LabelSerializer {
   private implicit val labelIdFormat: Format[LabelId] = Json.valueFormat[LabelId]
-  private implicit val unparsedLabelIdFormat: Format[UnparsedLabelId] = Json.valueFormat[UnparsedLabelId]
+  private implicit val unparsedLabelIdReads: Reads[UnparsedLabelId] = Json.valueReads[UnparsedLabelId]
+  private implicit val unparsedLabelIdWrites: Writes[UnparsedLabelId] = Json.valueWrites[UnparsedLabelId]
```

Comment:
Why not leaving Format here if there is no particular process for serialization/deserialization?

Reply:
I specifically use `unparsedLabelIdReads` below:

```scala
  private implicit val labelMapUpdateRequestReads: Reads[Map[UnparsedLabelId, LabelPatchObject]] =
    Reads.mapReads[UnparsedLabelId, LabelPatchObject] { string => unparsedLabelIdReads.reads(JsString(string)) }
```